modules/tile-converter/src/lib/utils/write-queue.ts

```diff
@@ -1,4 +1,7 @@
 import {Queue} from './queue';
+import process from 'process';
+
+const MEMORY_LIMIT = 4 * 1024 * 1024 * 1024; // 4Gb
```

Comment:
Maybe a short comment on why this is a good limit?

Also is it gigabit (Gb) or gigabyte (GB)>

Reply:
I'm not sure it is so good. We tested it and it worked well.

Reply:
Then add a comment to say that this is based on testing.

Reply:
Done